ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.scss

```diff
@@ -23,11 +23,6 @@
         flex-direction: column;
         align-items: center;
 
-        // There is no other way to hide it but to patch the yasqe
-        .yasqe_share {
```

Comment:
Why did you remove this? How did you replaced it?

Reply:
This is a mistake, when I merge the conflicts after rebasing. I will revert it.

Reply:
Done